pkg/storage/minio/deleter.go

```diff
@@ -12,9 +12,11 @@ func (v *storageImpl) Delete(ctx context.Context, module, version string) error
 	const op errors.Op = "minio.Delete"
 	sp, ctx := opentracing.StartSpanFromContext(ctx, "storage.minio.Delete")
 	defer sp.Finish()
-	if !v.Exists(ctx, module, version) {
+	exists, err := v.Exists(ctx, module, version)
```

Comment:
exists will be false with as a default value if error is set

Reply:
0e7477b